### PR TITLE
Add link to examples repo on JWT and SAML authentication

### PR DESCRIPTION
**Pull request by dragonsahead**
Adding links to the examples repository to make it easier for our users to find them

### CODE REVIEW

File: docs/enterprise-guide/authenticating-with-jwt.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ Once you have configured your JWT authentication, you can choose to disable the
 
 ![Password disable](images/password-disable.png)
 
+### Examples
+
+You can find examples about JWT Authentication in the [SSO examples repository](https://github.com/metabase/sso-examples)
```
Comment (jeff-bruemmer) [inline docs/enterprise-guide/authenticating-with-jwt.md:43]:
```suggestion
### Example code using JWT

You can find example code that uses JWT authentication in the [SSO examples repository](https://github.com/metabase/sso-examples).
```

File: docs/enterprise-guide/authenticating-with-saml.md [hunk 2]
```diff
@@ -190,6 +190,10 @@ Once you have configured SAML authentication, you can choose to disable the opti
 
 When users log in to Metabase for the first time via SSO, this will automatically create a Metabase account for them, which will trigger an email notification to Metabase administrators. If you don't want these notifications to be sent, you can turn this toggle off at the bottom of the Authentication page.
 
+### Examples
+
+You can find examples about SAML Authentication in the [SSO examples repository](https://github.com/metabase/sso-examples)
```
Comment (jeff-bruemmer) [inline docs/enterprise-guide/authenticating-with-saml.md:195]:
```suggestion
### Example code using SAML

You can find example code that uses SAML authentication in the [SSO examples repository](https://github.com/metabase/sso-examples).
```

**Review by jeff-bruemmer — APPROVED** [review]
Just some suggestions for rephrasing.

Thanks for adding links to the example repo, @paoliniluis.

**Comment by paoliniluis** [comment]
:rocket: :rocket: :rocket: @jeff-bruemmer :rocket: :rocket: :rocket:

**Pull request merged**